packages/commonwealth/server/controllers/server_groups_methods/update_group.ts

```diff
@@ -89,8 +108,52 @@ export async function __updateGroup(
         transaction,
       });
     }
+
     // update group
     await group.update(toUpdate, { transaction });
+
+    if (topicsToAssociate && topicsToAssociate.length > 0) {
+      // add group to all specified topics
+      await this.models.Topic.update(
+        {
+          group_ids: sequelize.fn(
+            'array_append',
+            sequelize.col('group_ids'),
+            group.id
+          ),
+        },
+        {
+          where: {
+            id: {
+              [Op.in]: topicsToAssociate.map(({ id }) => id),
+            },
+          },
+          transaction,
+        }
+      );
```

Comment:
This allows duplicate group IDs to be added to `group_ids`, so the array might look like `[1, 2, 3, 4, 4, 4]`.

The code should ensure that no duplicate IDs are added.

Reply:
There is a code section below that which removed duplicate ids, also I am not able to reproduce


https://github.com/hicommonwealth/commonwealth/assets/51641047/22892c92-ba71-4907-b8fe-40adc30f57eb

Reply:
In the `topics` array, you're probably passing in a group that doesn't exist.

Try a group that exists. E.g. if the group ID is 1234, then set `topics` to `1234`, and invoke the PUT request multiple times.

Reply:
i see, i don't get the dups (lets say `1463`) but I do get dups in group ids. Will fix that


https://github.com/hicommonwealth/commonwealth/assets/51641047/f988fb96-5157-4e7e-804f-c5f691dea270

Reply:
Right, that's what I meant 👍

Reply:
Fixed [here](https://github.com/hicommonwealth/commonwealth/pull/5499/commits/ba0b93468e64b57b3c883137dc769f25d2eaede1)